packages/jest-cli/src/runJest.js

```diff
@@ -84,11 +84,13 @@ const runJest = async (
 
     if (
       data.paths.length === 1 &&
-      config.silent !== true &&
-      config.verbose !== false
+      hasteContext.config.silent !== true &&
```

Comment:
That is happening somewhere in the larger PR, yes. It creates more merge conflicts if I do it in this one.